client/src/game/ui/tools/DiceTool.vue

```diff
@@ -70,157 +112,584 @@ function addSelector(selector: (typeof DxConfig.selectorOptions)[number]): void
+                            :key="i"
+                            class="dice-history-entry-wrapper"
+                        >
+                            <div
+                                class="dice-history-entry"
+                                :class="{ 'highlighted-history-entry': showHistoryBreakdownFor === i }"
+                                title="Toggle Breakdown"
+                                @click="showHistoryBreakdownFor = showHistoryBreakdownFor === i ? null : i"
+                            >
+                                <div class="history-grid-user">{{ player }}</div>
+                                <div class="history-grid-input">{{ name }}</div>
+                                <div class="history-grid-result">{{ historyRoll.result }}</div>
+                                <div class="toggle-chevron">
+                                    <font-awesome-icon :icon="showHistoryBreakdownFor === i ? 'chevron-up' : 'chevron-down'" />
+                                </div>
+                            </div>
+                            <div class="drawer-transition-wrapper" style="margin:0">
+                                <Transition name="drawer-expand" @after-enter="scrollToHistoryEntry">
+                                    <div v-if="showHistoryBreakdownFor === i" class="history-breakdown">
+                                        <button class="reroll-button" @click="populateInputFromHistoryIndex(i)">{{ t('game.ui.tools.DiceTool.reroll') }}</button>
+                                        <div class="breakdown-scroll-container">
+                                            <div class="breakdown-flex-container">
+                                                <div
+                                                    v-for="[index, part] of historyRoll.parts.entries()"
+                                                    :key="index"
+                                                >
+                                                    <div v-if="part.longResult" class="dice-result">
+                                                        <div class="input">{{ part.input ?? "" }}</div>
+                                                        <div class="ops">{{ '(' + part.longResult.replaceAll(',', ' + ') + ')' }}</div>
+                                                        <div class="value">{{ part.shortResult }}</div>
+                                                    </div>
+                                                    <div v-else-if="part.shortResult === '+' || part.shortResult === '-'" class="operator-result">
+                                                        {{ part.shortResult }}
+                                                    </div>
+                                                    <div v-else class="literal-result">
+                                                        {{ part.shortResult }}
+                                                    </div>
```

Comment:
This is completely repeated from the main result box if I'm not mistaken.

Might be worth to move this out to a separate component then (e.g. `DiceResults.vue`)